apps/molgenis-components/src/components/tables/TableExplorer.vue

```diff
@@ -739,13 +712,13 @@ export default {
       return {
         showColumns: ['name'],
         showFilters: ['name'],
-        conditions: {"name": ["pooky", "spike"]},
+        conditions: {"name": "pooky,spike"},
```

Comment:
this looks like the public filter api is changed, correct ? or is the a bug fix

Reply:
Neither, what goes into the filter is still the same, just the interface of this component has changed.

Reply:
that what i mean by api change , if we move from key->array to key->comma separated list we need to change all the callers , why this change using a array of conditions seams better to me

Reply:
We used moved around functionality, what goes in now is the conditions string of the URL. It now gets parsed inside this component, due to the parent not having access to the column types. Will update the name of the property to reflect this.